lib/tasks/change_note.rake

```diff
@@ -0,0 +1,37 @@
+namespace :change_note do
+  desc "List change notes for a content ID"
+  task :list, %i[content_id] => :environment do |_, args|
+    change_history = Document
+      .find_by(content_id: args[:content_id])
+      .editions
+      .where(minor_change: false)
+      .pluck(:id, :major_change_published_at, :change_note)
+
+    change_history.each do |change|
```

Comment:
v minor comment:
If you put in the wrong `content_id` and get an empty array for `change_history`, there won't be any output right? Which I guess could be a bit confusing. Maybe  `if change_history.empty? puts "no change notes found" ` kind of thing might be nice?

Reply:
I'll add an output for when there's no history.  That should make this clearer.